src/base/http/connection.cpp

```diff
@@ -161,15 +161,15 @@ bool Connection::acceptsGzipEncoding(QString codings)
         return false;
     };
 
-    const QVector<QStringRef> list = codings.remove(' ').remove('\t').splitRef(',', Qt::SkipEmptyParts);
+    const QList<QStringView> list = QStringView(codings.remove(' ').remove('\t')).split(QChar::fromLatin1(','), Qt::SkipEmptyParts);
```

Comment:
We can't (and we shouldn't even try) decide to use only one of these options, since they are actually different things. True, in some contexts we can still use any of them, but in general this is not the case.

Reply:
I meant in the context of all options are available then we should prefer one path.

Off topic, I checked the conversion path: `QChar(char16_t ch)` is basically copy which is fast and good. And I thought maybe we could add `u` prefix to most strings and use `QString::fromUtf16` and reap the same benefits but that function turns out to be expensive... Another possibility is using `QString(const QChar *unicode, int size = -1)` (which only do some trivial checking) and cast the `u"str"` to `QChar*`, it works fine on my machine but I'm not sure it is universally applicable to all different architectures.

Reply:
>I meant in the context of all options are available then we should prefer one path.

Of course.
We use QLatin1String unless other is required.

Reply:
>I thought maybe we could add `u` prefix to most strings and use `QString::fromUtf16` and reap the same benefits but that function turns out to be expensive... Another possibility is using `QString(const QChar *unicode, int size = -1)` (which only do some trivial checking) and cast the `u"str"` to `QChar*`, it works fine on my machine but I'm not sure it is universally applicable to all different architectures.

It doesn't look like we can guarantee its portability (at least, without an extra headache for us).
In addition, we don't really need to create QString in many cases where we use QLatin1String.